js/weather.js

```diff
@@ -0,0 +1,219 @@
+    else if (weather === "Rain"){
+        img.src = "https://png.icons8.com/ios/50/000000/rain.png";
+    }
+    else if(weather === "Snow"){
+        img.src = "https://png.icons8.com/ios/50/000000/winter.png";
+    }
+    else if(weather === "Hail"){
+        img.src = "https://png.icons8.com/ios/50/000000/hail.png";
+    }
+    else if(weather == "Haze"){
+        img.src = "https://png.icons8.com/ios/50/000000/fog-day.png";
+    }
+    else {
+        img.src = "https://png.icons8.com/ios/50/000000/sun.png";
+    }
+
+    */
+
+    // appends message to article
+    content.appendChild(node); // adds temperature
+    content.appendChild(degree);
+    content.appendChild(sign); // celsius sign
+    content.appendChild(br); // adds break
+    content.appendChild(img); // appends logo
+    content.appendChild(space); // adds blank space
+    content.appendChild(weatherText); // weather info
+
+    // appends divs and set id
+    child.appendChild(content);
+    elementToAdd.appendChild(child);
+    elementToAdd.id = "weatherObject";
+
+    /*
+    // creates a button
+    var celsius = document.createElement("a");
+    celsius.onclick = convert(x, "C")
+    celsius.className = "button";
+    celsius.classList.add("is-primary");
+    celsius.classList.add("is-outlined");
+    celsius.textContent = "C";
+    celsius.id = "celsius";
+
+    // creates a button
+    var farenheit = document.createElement("a");
+    farenheit.onclick = convert(x, "F");
+    farenheit.className = "button";
+    farenheit.classList.add("is-primary");
+    farenheit.classList.add("is-outlined");
+    farenheit.textContent = "F";
+    farenheit.id = "fahrenheit";
+
+    // creates new row
+    var newDiv = document.createElement('div');
+    newDiv.appendChild(celsius);
+    newDiv.appendChild(farenheit);
+
+    elementToAdd.appendChild(newDiv);    
+    */
+
+    // appends article to container
+    document.getElementById("contain").appendChild(elementToAdd);
+
+}
+
+function convert (x, text){
+    var final = x - 273.15;
+    if (text == "F") {
+        final = final * (9/5);
+        final += 32;
+    }
+    return final;
+}
+
+function removeWeatherData (x) {
+    var elementToRemove = document.getElementById(x);
+    if(elementToRemove){
+        elementToRemove.parentNode.removeChild(elementToRemove);
+    }
+}
+
+function getWeatherData (url) {
+    fetch(url).then(function(response){
+        return response.json();
+    }).then(function(weatherData){
+        var temp = weatherData.main.temp;
+        var weather = weatherData.weather[0].main;
+        // var description = weatherData.weather[0].description;
+        temp = convert(temp);
+        temp = temp.toFixed();
+
+        if(document.getElementById("weatherObject")){
+            var id1 = "weatherObject";
+            var id2 = "celsius";
+            var id3 = "fahrenheit";
+
+            removeWeatherData(id1);
+            removeWeatherData(id2);
+            removeWeatherData(id3);
+        }
```

Comment:
You can use direct string declaration. e.g `removeWeatherData("weatherObject")`.

Method call for `id2` and `id3` seems redundant as I found no element with said `id`?